cmd/api-router.go

```diff
@@ -285,6 +285,9 @@ func registerAPIRouter(router *mux.Router) {
 		// GetObjectLegalHold
 		router.Methods(http.MethodGet).Path("/{object:.+}").HandlerFunc(
 			collectAPIStats("getobjectlegalhold", maxClients(gz(httpTraceAll(api.GetObjectLegalHoldHandler))))).Queries("legal-hold", "")
+		// GetObject - note gzip compression is *not* added due to lambda functions
```

Comment:
You can add it. it is off by default, and enabled through 'api' config.

Seems like I forgot to update the comment below.